library/core/src/ops/range.rs

```diff
@@ -979,6 +979,88 @@ impl<T> RangeBounds<T> for RangeToInclusive<&T> {
     }
 }
 
+/// Used to convert a range into start and end bounds, consuming the
+/// range by value.
+///
+/// `IntoBounds` is implemented by Rust’s built-in range types, produced
+/// by range syntax like `..`, `a..`, `..b`, `..=c`, `d..e`, or `f..=g`.
+#[unstable(feature = "range_into_bounds", issue = "136903")]
+pub trait IntoBounds<T: Sized>: RangeBounds<T> {
```

Comment:
```suggestion
pub trait IntoBounds<T>: RangeBounds<T> {
```

`<T>` means `<T: Sized>`